packages/next/src/build/analysis/get-page-static-info.ts

```diff
@@ -142,6 +145,7 @@ function checkExports(
       let generateSitemaps: boolean = false
       let generateStaticParams = false
       let extraProperties = new Set<string>()
+      let directives = new Set<string>()
 
       for (const node of swcAST.body) {
```

Comment:
not for this PR and mostly curiosity q, have we considered moving this into next-swc instead of calling `parseModule` and traverse ast in JS?

Reply:
yeah I think we should move there if possible